pom.xml

```diff
@@ -90,6 +99,14 @@
                     <sourceDirectories>src</sourceDirectories>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.liquibase</groupId>
+                <artifactId>liquibase-maven-plugin</artifactId>
+                <version>4.24.0</version>
+                <configuration>
+                    <propertyFile>src/main/resources/liquibase.properties</propertyFile>
+                </configuration>
+            </plugin>
```

Comment:
Not sure that this dependencies nedded for pagination or swagger.

Reply:
do we still need it? if not - let's remove it